unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -949,8 +949,8 @@ loop = do
 
                 fixupOutput :: Path.HQSplit -> HQ.HashQualified Name
                 fixupOutput = fmap Path.toName . HQ'.toHQ . Path.unsplitHQ
-            NamesI thing -> do
-              ns0 <- basicParseNames
+            NamesI global thing -> do
+              ns0 <- if global then pure basicPrettyPrintNames else basicParseNames
```

Comment:
`NamesI` picks up a `IsGlobal` flag, which is used to control which scope names to use (the local parsing names, or the global pretty print names).

Maybe `basicPrettyPrintNames` should be renamed to `globalPrettyPrintNames` and `basicParseNames` to `localParseNames`?